bindings/pybind11/tests/test_idyntree_core.py

```diff
@@ -236,6 +123,17 @@ def test_creation(self):
     for i in range(3):
       self.assertEqual(transform.position[i], position[i])
 
+  def test_creation_from_numpy(self):
+    position = np.array([1., 2., 3.])
+    rotation = np.array([[0., 0., 1.], [1., 0., 0.], [0., 1., 0.]])
+    transform = iDynTree.Transform(rotation, position)
```

Comment:
This is pending the other discussion

Reply:
which one?

Reply:
The one on implicit vs explicit conversion